src/BUILD

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility=["//visibility:public"])
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+load("@npm_bazel_jasmine//:index.bzl", "jasmine_node_test")
+
+ts_library(
+    name = "shadowsocks_config",
+    srcs = ["shadowsocks_config.ts"],
```

Comment:
do you actually get benefits from using Bazel at such a small scope?

Reply:
Not much. The main reason is to:

1. Make sure I'm doing the right thing, so I can apply the same in the bigger repos
1. Make sure CI works
1. Make it easier to consume this repo in outline-server that I'm converting to Bazel.